ui/desktop/app/routes/scopes/scope/projects/targets/target/hosts.js

```diff
@@ -27,9 +27,7 @@ export default class ScopesScopeProjectsTargetsTargetHostsRoute extends Route {
     );
 
     // Extract host ids from all host sets
-    const hostIds = hostSets.map(({ host_ids }) =>
-      host_ids.content.map((hostId) => hostId.value)
```

Comment:
What was the `content` here for previously? When I was testing it's just a normal array of host IDs now but I didn't expect the `content`